src/Entity/ContentItem.php

```diff
@@ -10,6 +10,7 @@
 
 /**
  * Base class for ContentItem entities.
+ * @method setPage(?Page $page)
```

Comment:
Zou deze er niet gewoon als `abstract public function setPage(?Page $page);` in kunnen (of moeten)?

Reply:
Kan. Nog beter zou overigens zijn `setPage(PageInterface $page)` maar dat breekt de huidige implementatie in alle projecten sowieso.

Reply:
"Breekt" het nu dan al niet omdat `?Page` een `Zicht\Bundle\PageBundle\Entity\Page` is en in projecten het een `App\Entity\Page` is?

Reply:
Nee dat lijkt goed te gaan [https://github.com/zicht/cookiecutter-symfony-cms/pull/651/files#diff-6991d70b7b687ca2be4dc74060d7a634664a7d236c02f95f2a30dc0439abbd9dR53](https://github.com/zicht/cookiecutter-symfony-cms/pull/651) :slightly_smiling_face: